### ISSUE TITLE
Warnings when running resample_data()

### ISSUE THREAD

**Issue by anetasie** [sherpa/sherpa#723]
I see a lot of warnings while running 'resample_data()' in CIAO4.12 Sherpa. Test data for 
asymmetric errors will show the screen printout for every iteration. 

      load_ascii_with_errors(1,'gro.txt')
      set_stat('leastsq')
      set_model(powlaw1d.p1)
      fit()
      sample=resample_data(1,1000)

  

       .... warnings.warn("Converting array {} to numpy array".format(array))
     /Users/aneta/soft/ciao-4.12/ciao-4.12/lib/python3.5/site-packages/sherpa-4.12.0-py3.5- 
     macosx-10.12-x86_64.egg/sherpa/data.py:49: UserWarning: Converting array 
     [13.381319496084906, 13.633260235068068, 14.306213914726577, 13.983557790201848, 
    13.171125623509718, 12.488433532904432, 14.626279812074497, 16.012878283581944, 
    15.258566424800357, 13.690025984816309, 18.514528963412666, 14.876755444565106, 
    16.32593948807569, 11.884746992186537, 11.257004828977017, 11.134748193051466, 
    11.666966554920794, 11.688190461005217, 9.080282498957114, 10.042537574980583, 
    9.794066377984949, 9.180452401453762, 10.331401797264734, 10.69460004113246, 
    8.958842965993878, 7.552101436170255, 7.858390587996477, 10.645408597899696, 
    8.371732877410645, 6.9382374151118675, 8.068750514358847, 5.725444743683091, 
    6.479597857237468, 7.947197940606003, 5.017390444344825, 4.408614337830014, 
    5.470251340952552, 5.400760418773997, 4.926646231399067, 6.669486210793662, 
    3.946138706536805, 5.491861097640214, 20.473864743135888, 9.584644867183084, 
    2.9941695735462095, 3.8880767097610125, 2.8402196610798445, 1.6563930690598663, 
    2.5479535666395163, 2.3527045444059103, 2.774612664648795, 2.2056629549101965, 
    3.6670732010833658, 2.9395946286845893, 3.4041964024021647, 2.261442059019056, 
    1.537098568238905, 4.384837877801289, 0.5012254710457373, 4.472802659515596, 1 
    1.9897275999338797] to numpy array
     warnings.warn("Converting array {} to numpy array".format(array))
     p1.gamma : avg = -0.46824871286051334 , std = 0.07754346661186172
     p1.ampl : avg = 176.7461287858567 , std = 63.422462767221504

**Comment by dtnguyen2** [comment]
I think I have a fix for this: 
```
(sherpa) [dtn@devel12 sherpa]$ git status
On branch fix_issue_723
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   sherpa/sim/__init__.py

no changes added to commit (use "git add" and/or "git commit -a")
```
so when I run the script, the warning msgs are gone:
```
(sherpa) [dtn@devel12 warning]$ python tmp.py 
WARNING: imaging routines will not be available, 
failed to import sherpa.image.ds9_backend due to 
'RuntimeErr: DS9Win unusable: Could not find ds9 on your PATH'
Dataset               = 1
Method                = levmar
Statistic             = leastsq
Initial fit statistic = 1.82337e+08
Final fit statistic   = 202.862 at function evaluation 61
Data points           = 61
Degrees of freedom    = 59
Change in statistic   = 1.82337e+08
   p1.gamma       -0.525284    +/- 0.0196587   
   p1.ampl        224.17       +/- 25.3449     
p1.gamma : avg = -0.4637491345322396 , std = 0.07918121910527111
p1.ampl : avg = 173.0752152227821 , std = 64.36640293121916
```

**Comment by DougBurke** [comment]
Closed by #725
